SimG4Core/CustomPhysics/plugins/RHStopTracer.cc

```diff
@@ -14,12 +14,14 @@
 #include "G4Run.hh"
 #include "G4Event.hh"
 #include "G4SystemOfUnits.hh"
+#include "G4ParticleTable.hh"
+#include "G4ParticleDefinition.hh"
 
 RHStopTracer::RHStopTracer(edm::ParameterSet const & p) {
   edm::ParameterSet parameters = p.getParameter<edm::ParameterSet>("RHStopTracer");
   mStopRegular = parameters.getUntrackedParameter<bool>("stopRegularParticles", false);
-  mTraceEnergy = 1000 * parameters.getUntrackedParameter<double>("traceEnergy", 1.e20); // GeV->KeV
-  mTraceParticleNameRegex = parameters.getParameter<std::string>("traceParticle");
+  mTraceEnergy = parameters.getUntrackedParameter<double>("traceEnergy", 1.e20); 
```

Comment:
@davidlange6 , G4 default unit is MeV, because this parameter of the class is used only to compare with G4 value, no conversion is needed. To clarify value a LogInfo printout is added, so user will see what is the value/unit of this energy.